tests/lanczos_simpleop.templates.hpp

```diff
@@ -0,0 +1,51 @@
+/*************************************************************************
+ * Copyright (c) 2017-2019 by the mfmg authors                           *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * This file is part of the mfmg libary. mfmg is distributed under a BSD *
+ * 3-clause license. For the licensing terms see the LICENSE file in the *
+ * top-level directory                                                   *
+ *                                                                       *
+ * SPDX-License-Identifier: BSD-3-Clause                                 *
+ *************************************************************************/
+
+#ifndef MFMG_LANCZOS_SIMPLEOP_TEMPLATE_HPP
+#define MFMG_LANCZOS_SIMPLEOP_TEMPLATE_HPP
+
+#include <cassert>
+
+#include "lanczos_simpleop.hpp"
```

Comment:
Consistency: Use angle brackets.

Reply:
Hmm, I tried that, but it does not find the header if I switch to angle brackets...

Reply:
Yeah, the headers that are only in `tests` are not exposed correctly. This is the correct way to do it.